lib/install/management/appliance.go

```diff
@@ -91,6 +91,24 @@ func (d *Dispatcher) isVCH(vm *vm.VirtualMachine) (bool, error) {
 	return false, nil
 }
 
+func (d *Dispatcher) isContainerVM(vm *vm.VirtualMachine) (bool, error) {
+	if vm == nil {
```

Comment:
Is it possible that anybody pass a nil value here? Asking, because of excessive check is actually bad for readability and maintainability.